src/CONFIG/pyproject.toml

```diff
@@ -0,0 +1,24 @@
+[tool.poetry]
+name = "qat-config"
+version = "1.0.0"
+description = ""
+authors = [
+    "Hamid El Maazouz <helmaazouz@oxfordquantumcircuits.com>",
+    "Harry Waring <hwaring@oxfordquantumcircuits.com>",
+    "Jamie Friel <jfriel@oxfordquantumcircuits.com>",
+    "John Dumbell <jdumbell@oxfordquantumcircuits.com>",
+    "Kajsa Eriksson Rosenqvist <keriksson.rosenqvist@oxfordquantumcircuits.com>",
+    "Norbert Deak <ndeak@oxfordquantumcircuits.com>",
+    "Owen Arnold <oarnold@oxfordquantumcircuits.com>",
+]
+#readme = "README.md"
```

Comment:
Remove commented stuff (for now).

Reply:
I was actually going to get this replaced with an actual README as that shows up on the PyPI page.